front/components/DataGrid.js

```diff
@@ -197,6 +207,33 @@ export default class DataGrid extends React.Component {
         return pagesList;
     }
 
+    // Work in Progress
```

Comment:
we shouldnt merge commantated code, and we shouldnt merge `@wip` or `@todo`  tags

either we put it in another branch and work on it, either we remove this code

Reply:
💅